### DISCUSSION TITLE
Anthropic API?

### DISCUSSION THREAD

**Discussion by ItsRealDennis** [open-webui/open-webui#6503] [Q&A] [answered]
I saw someone using claude, how?

**Comment by kasxi** [comment, marked as answer]
Click on Workspace in the UI
Select Functions , then click Discover a Function
Search for Anthropic and copy the code
Click on Add Function
Enter a Name and Description , then click Save
Click the Settings button and add your API Key
Anthropic models will now appear in your model list

**Reply by ItsRealDennis** [reply]
Thanks! 

But when I get to "Enter a Name and Description, then click Save" I can only pick from openAI models. Is that correct?

**Reply by kasxi** [reply]
If you've added the anthropic function it should look like this in your functions list.
Click the gear icon to set your api key & then make sure the slider is on (green on mine in dark mode)
The anthropic models should then show up in your models list.

![image](https://github.com/user-attachments/assets/82d48e99-93b7-4305-bcc6-1b48a130b0bb)

**Reply by ItsRealDennis** [reply]
Ah, Thanks alot!

**Reply by metasikander** [reply]
to expand on this if any one else has the same issue as me trying this out...
when you get to the site after clicking "Discover a Funtion", select Function in the drop-down when searching. That way you don't try to import the models as i did :sweat_smile: 
![image](https://github.com/user-attachments/assets/6f456881-c6d4-4426-928e-bd492be96a20)
